README.md

```diff
@@ -67,7 +67,6 @@ simple 3D example.
 git clone --recursive https://github.com/blackwer/baobzi.git
 cd baobzi
 mkdir build
-cd build
```

Comment:
I never use the `cmake --build` stuff, so you actually need to `cd` into build here.